### PR TITLE
Refactoring GHA Editor

### PR DESCRIPTION
**Pull request by kolioOtSofia**
Closes: #Issue

### CODE REVIEW

**Review by Toalaah — APPROVED** [review]
lgtm

**Review by benjamin-frost — APPROVED** [review]

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+An ATDDTestingImage is a second image that is being opened in order to run tests in parallel. To make this seamless to the user the image is opened headless.
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/README.md:1]:
```suggestion
An ATDDTestingImage is a second image that is being opened in order to run tests in parallel. To make this seamless to the user the image is opened headlessly.
```

Comment (SebastianWagner2) [reply packages/AutoTDD-Core.package/ATDDTestingImage.class/README.md:1]: Somehow this same change is also on another branch

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process..st [hunk 2]
```diff
@@ -0,0 +1,4 @@
+accessing
+process: anOSProcess
+
+	Process := anOSProcess
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process..st:4]:
```suggestion
accessing
process: anOSProcess

	process := anOSProcess
```

Changes on another branch already

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process.st [hunk 3]
```diff
@@ -0,0 +1,4 @@
+accessing
+process
+
+	^ Process
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/process.st:4]:
```suggestion
accessing
process

	^ process
```

File: packages/AutoTDD-GUI.package/ATDDGitHubActionEditor.class/instance/makesValidRequest.st [hunk 4]
```diff
@@ -1,11 +1,8 @@
 testing
```
Comment (SebastianWagner2) [inline packages/AutoTDD-GUI.package/ATDDGitHubActionEditor.class/instance/makesValidRequest.st:1]:
```suggestion
utility
```

File: packages/AutoTDD-Core.package/ATDDTestingImage.class/class/startServer.st [hunk 5]
```diff
@@ -0,0 +1,9 @@
+initialization
+startServer
+
+	| server |
+	
+	server := WebServer new.
+	server listenOn: 8888;
+		addService: '/' action: [ :req | req send200Response: 'hello World'].
```
Comment (SebastianWagner2) [inline packages/AutoTDD-Core.package/ATDDTestingImage.class/class/startServer.st:8]:
```suggestion
		addService: '/' action: [ :req | req send200Response: 'Hello World'].
```

**Review by SebastianWagner2 — CHANGES_REQUESTED** [review]

**Review by SebastianWagner2 — APPROVED** [review]

**Pull request merged**